package com.twobit.driver.data.bluetooth

import android.annotation.SuppressLint
import android.bluetooth.BluetoothDevice
import com.twobit.driver.domain.bluetooth.BluetoothDeviceDomain

@SuppressLint("MissingPermission")
fun BluetoothDevice.toBluetoothDeviceDomain(): BluetoothDeviceDomain {
    return BluetoothDeviceDomain(
        name = name,
        address = address
    )
}